src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -115,20 +113,17 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
     public isEnabled(type: BannerType): boolean {
         switch (type) {
             case BannerType.InsidersNotebookSurvey:
-                if (this.useVSCodeNotebookEditorApi && this.applicationEnvironment.channel === 'insiders') {
+                if (this.applicationEnvironment.channel === 'insiders' && this.useVSCodeNotebookEditorApi) {
                     return this.isEnabledInternal(type);
                 }
                 break;
             case BannerType.ExperimentNotebookSurvey:
-                if (
-                    this.applicationEnvironment.channel === 'stable' &&
-                    this.experimentService.inExperiment(Experiments.NativeNotebook)
```

Comment:
Aah so the bug was we were not `awaiting` on the promise.

Reply:
yeah, I didn't notice it was a promise, anyway, its cleaner to just use `useVSCodeNotebookEditorApi`